packages/ckeditor5-image/ckeditor5-metadata.json

```diff
@@ -7,7 +7,70 @@
 			"docs": "features/image.html#inserting-images-via-pasting-url-into-editor",
```

Comment:
The docs have been rewritten and the links have changed. I know that there are some new redirects added in the Umberto, but IMHO it would be nice to use valid links in the metadata files.

This concerns also other plugins in the `ckeditor5-image` package.